sdk/communication/communication-rooms/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.1.0 (2024-04-15)
+
+### Features Added
```

Comment:
I would remove this header too as we are not releasing any feature. Simply the put the following description under the new version 
"General Availability version of PSTN dial-out capability feature."